README.md

```diff
@@ -1,20 +1,20 @@
 # Google Maps React Hooks
-[![npm version](https://img.shields.io/npm/v/@ubilabs/google-maps-react-hooks)](https://www.npmjs.com/package/@ubilabs/google-maps-react-hooks) [![GitHub license](https://img.shields.io/badge/license-MIT-green.svg)](https://github.com/ubilabs/google-maps-react-hooks/blob/develop/LICENSE)
+[![npm version](https://img.shields.io/npm/v/@ubilabs/google-maps-react-hooks)](https://www.npmjs.com/package/@ubilabs/google-maps-react-hooks) [![GitHub license](https://img.shields.io/badge/license-MIT-green.svg)](./LICENSE)
 
 ## Description
 
 This is a JavaScript library to easily implement a Google Maps map into your React application. It comes with a collection of React hooks to access the Google Maps map instance all over your components and to handle some common interactions with the map.
 
 #### Table of contents
-- [Google Maps React Hooks Library](https://github.com/ubilabs/google-maps-react-hooks/blob/main/library/README.md)
-- [Examples](https://github.com/ubilabs/google-maps-react-hooks/tree/main/examples)
-  - [Basic Google Map](https://github.com/ubilabs/google-maps-react-hooks/blob/main/examples/basic-google-map)
-  - [Google Map with Markers](https://github.com/ubilabs/google-maps-react-hooks/blob/main/examples/google-map-with-markers)
-  - [Places Autocomplete](https://github.com/ubilabs/google-maps-react-hooks/blob/main/examples/places-autocomplete)
-  - [Directions Service](https://github.com/ubilabs/google-maps-react-hooks/blob/main/examples/directions)
-  - [Geocoding Service](https://github.com/ubilabs/google-maps-react-hooks/blob/main/examples/geocoding)
-  - [Places Library](https://github.com/ubilabs/google-maps-react-hooks/blob/main/examples/places)
-  - [Distance Matrix Service](https://github.com/ubilabs/google-maps-react-hooks/blob/main/examples/distance-matrix)
+- [Google Maps React Hooks Library](./library)
```

Comment:
This not linking to the `README.md` but to the director itself.

```suggestion
- [Google Maps React Hooks Library](./library/README.md)
```

Same for all the links below...

Reply:
This was intended. If you open a directory that includes a README file, GitHub displays the content of the README right below the content of the directory.

Reply:
Okay, thought it should just navigate between the READMEs. But then all good